packages/web-pkg/src/components/Editor.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div id="editor" ref="toastuiEditorRef" :data-markdown-mode="isMarkdown"></div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, unref, PropType, ref, onMounted, watch } from 'vue'
+import { Resource } from '@ownclouders/web-client/src/helpers/resource/types'
+
+import '@toast-ui/editor/dist/toastui-editor.css'
+import '@toast-ui/editor/dist/theme/toastui-editor-dark.css'
+import '@toast-ui/editor/dist/i18n/cs-cz'
```

Comment:
hm.... could you make a comment in the code location in the runtime where we define all the available languages? because it looks like we'd need to update this file here as well when adding or removing languages. Something like `Please also make language adjustments in packages/web-pkg/src/components/TextEditor.vaue` (or whatever name you'll choose for the component).